Snippets/Snippets_6/Transactions.cs

```diff
@@ -32,17 +50,17 @@ public void TransportTransactionsWithScope()
         {
             #region TransactionsWrapHandlersExecutionInATransactionScope
             BusConfiguration busConfiguration = new BusConfiguration();
-            busConfiguration.Transactions()
-                .WrapHandlersExecutionInATransactionScope();
+            busConfiguration.UnitOfWork()
+                .WrapHandlersInATransactionScope();
             #endregion
         }
 
-        public void CustomTransactionTimeout()
+        public void UnitOfWorkWrapHandlersInATransactionScope()
         {
-            #region CustomTransactionTimeout
+            #region UnitOfWorkWrapHandlersInATransactionScope
```

Comment:
Should we pull the UOW snippet out into a UnitOfWork.cs to make the split even more obvious?

And then perhaps rename `Transactions.cs` => `TransportTransactions.cs` ?